src/TouristInfoCards.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="cardContainer">
+      <div className="card">
+        <img
+          src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/16/Kingston_Bridge_in_Glasgow.jpg/800px-Kingston_Bridge_in_Glasgow.jpg"
+          className="card-img-top"
+          alt="Glasgow on a budget"
+        />
+        <div className="card-body">
+          <h5 className="card-title">Glasgow</h5>
+          <p className="card-text">
+            Glasgow's city centre is home to flagship stores, impressive
+            shopping centres and designer within an easily walkable area.
+          </p>
+          <a href="http://peoplemakeglasgow.com" className="btn btn-primary">
+            Visit Glasgow
+          </a>
+        </div>
+      </div>
```

Comment:
It looks like this set of HTML is repeated a few times - this could be extracted into a separate component and then rendered multiple times rather than repeating all this HTML